google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/BigtableBatchingCallSettings.java

```diff
@@ -145,9 +194,135 @@ public Builder setRetrySettings(@Nonnull RetrySettings retrySettings) {
       return this;
     }
 
+    /**
+     * Enables / disables latency based throttling. If enabling the setting, targetRpcLatency needs
+     * to be set.
+     */
+    public Builder setLatencyBasedThrottling(
```

Comment:
Here and everywhere else `BetaApi` for new public surfaces.

Also, can we split this up into enableLatencyBasedThrottling(targetMs) and disableLatencyBasedThrottling()?

Reply:
This class already has `BetaApi` annotation, will we still need them for each method?

Will split up the settings.